modules/mariadb/src/main/java/org/testcontainers/containers/MariaDBContainer.java

```diff
@@ -18,7 +18,7 @@ public class MariaDBContainer<SELF extends MariaDBContainer<SELF>> extends JdbcD
     private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("mariadb");
 
     @Deprecated
-    public static final String DEFAULT_TAG = "10.3.6";
+    public static final String DEFAULT_TAG = "10.3.39";
```

Comment:
Any issue is we keep 10.3.6 as a default? I would like to avoid breaking existing test that relies on default constructor.

Reply:
Yes in fact, because 10.3.6 does not have ARM image variants and crashes when you try to run on Colima under emulation as noted in https://github.com/testcontainers/testcontainers-java/pull/8131#issuecomment-1906393732

Reply:
10.3.39 with multiple archs: https://github.com/testcontainers/testcontainers-java/commit/3aa5fbfa17be09f4e42b2300396890f76f508c86#diff-2568f90aa8a2109ff614414c5ef2b547f6bd7e8b8d4db9e81130d091b3e5d67cR19

10.3.6 with only amd64

https://hub.docker.com/layers/library/mariadb/10.3.6/images/sha256-ab3aab60f76b6d2d2e2d9d2c486029b63ad7cc9517ad4567a27c66999f8a23ce?context=explore